### DISCUSSION TITLE
How can I round not only the corners of the button but also the corners of the "background behind the button"?

### DISCUSSION THREAD

**Discussion by oneLEAM** [TomSchimansky/CustomTkinter#1441] [Q&A] [answered]
How, along with the round corners of the button, to make round corners "background behind the button", when it is located on the frame, but overlaps another frame with its "background button".

An example of what I'm talking about:
https://imgur.com/a/UzYEbvA

**Comment by Akascape** [comment, marked as answer]
@oneLEAM 
All the ctk widgets are actually square in shape but the corner portions are camouflaged in such a way that it appears to be round in the frame.
(In short, bg_color of button is matched with the fg_color of frame)

In your case, you are using 2 frames overlapped but the button can only have 1 bg color by default. 

You have to manually adjust the four corner colors adding `background_corner_colors=(color1, color2, color3, color4)` in the button parameter.

**Reply by oneLEAM** [reply]
Thank you so much for your help!

**Comment by Akascape** [comment]
@oneLEAM A new solution for this issue: https://github.com/TomSchimansky/CustomTkinter/discussions/2214
